controllers/articles-controller.js

```diff
@@ -9,4 +9,13 @@ exports.getArticleById = (req, res, next) => {
     .catch((err) => {
         next(err);
     })
-}
\ No newline at end of file
+}
+
+exports.getArticles = (req, res, next) => {
+    selectArticles()
+    .then((articlesArr) => {
+        // currently returning articles array in DESC order, but no comment_count
```

Comment:
Please remove any comments and console logs before committing :)

Reply:
Woops I missed this, thank you! Also when I went to the endpoint.json I noticed that there was already an entry for GET /api/articles - do I have to write another one, or tweak this one? Yesterday eve I saw it and didn't read all of it, now I can see the description is slightly different than the functionality I've made.

Reply:
Good spot - yeah might just need some tweaking :)